File: src/server/AnkaCMS.Service/Models/PublicContentModel.cs
using System;
using AnkaCMS.Core.ValueObjects;

namespace AnkaCMS.Service.Models
{
    public class PublicContentModel
    {
        public string Code { get; set; }
        public string Name { get; set; }
        public string ShortName { get; set; }
        public string Description { get; set; }
       
        public string Keywords { get; set; }
        public string ContentDetail { get; set; }
        public string Url { get; set; }
        public string ImageName { get; set; }
        public string ImagePath { get; set; }
        public string ImageFileType { get; set; }
        public int ImageFileLength { get; set; }
        public DateTime LastModificationTime { get; set; }
        public IdCodeName Category { get; set; }
        public Guid ContentId { get; set; }
        public IdCodeName Language { get; set; }
    }
}

File: src/client/angular-web-public/src/app/models/public-content-model.ts
import { IdCodeName } from '../value-objects/id-code-name';

export class PublicContentModel {
    code: string;
    name: string;
    shortName: string;
    description: string;
    keywords: string;
    contentDetail: string;
    url: string;
    imageName: string;
    imagePath: string;
    imageFileType: string;
    imageFileLength: number;
    lastModificationTime: Date;
    category: IdCodeName;
    contentId: string;
    language: IdCodeName;
    constructor() {
        this.language = new IdCodeName();
        this.category = new IdCodeName();
    }
}
